src/cli/config.py

```diff
@@ -0,0 +1,136 @@
+"""this implements controllers and hooks to connect to
+managed_data.py"""
+
+import os
+import shutil
+
+import cement
+import yaml
+from deeporigin.config import CONFIG_YML_LOCATION, TEMPLATE, get_value
+from deeporigin.exceptions import DeepOriginException
+from deeporigin.utils import _print_dict
+
+
+class ConfigController(cement.Controller):
+    """Controller for data subcommand of CLI"""
+
+    class Meta:
+        label = "config"
+        stacked_on = "base"
+        stacked_type = "nested"
+        help = "Show and modify configuration"
+        description = """
+Show and modify configuration file to connect to Deep Origin 
+            """
+
+    def _default(self):
+        self.app.pargs.json = False
+        self.show()
+
+    @cement.ex(
```

Comment:
Another small function that would be helpful to add is a function that prints out the path to the user configuration file.